src/mmrelay/main.py

```
@@ -62,6 +63,82 @@
 
 # Flag to track if banner has been printed
 _banner_printed = False
+_ready_file_path = os.environ.get(
+    "MMRELAY_READY_FILE", "/tmp/ready"
```

Ruff (0.14.14)

[error] Probable insecure usage of temporary file or directory: "/tmp/ready"

(S108)